app/src/components/metamine/visualizationNU/histogram.vue

```diff
@@ -1,10 +1,9 @@
 <template>
-    <div ref="histogramPlot"></div>
+  <div ref="histogramPlot"></div>
 </template>
 
 <script>
```

Comment:
I'm not sure exactly where the source of the problem is, but for the histogram plots the x axis labels do not update properly when the dropdown is changed

Reply:
That's weird. I tested on my end and it's working

Reply:
Here's what shows up for me locally when I change the x axis from C66 to C12 on the histogram. Noticing on both Chrome and Firefox
<img width="389" alt="Screenshot 2023-12-23 at 12 31 23 AM" src="https://github.com/Duke-MatSci/materialsmine/assets/24441706/7f38498c-f228-4b14-91d8-615d46421767">

Reply:
Got it! Fixed it.